pages/_app.js

```diff
@@ -5,17 +5,24 @@ import { uid } from "uid";
 
 export default function App({ Component, pageProps }) {
   const [shoppingItems, setShoppingItems] = useState(shoppingItemsData);
+  const [isDelete, setIsDelete] = useState();
```

Comment:
```suggestion
  
```
I think you don't need this useState, right?

Reply:
Forgot that, thanks!